soxr-sys/src/lib.rs

```diff
@@ -24,8 +26,23 @@ mod tests {
 
         use hound::{WavReader, WavSpec, WavWriter};
 
-        let input_wav_path = "input.wav";
-        let output_wav_path = "output.wav";
+        let input_wav_path = "test-input.wav";
+        let output_wav_path = "test-output.wav";
+
+        let spec = WavSpec {
+            channels: 1,
+            sample_rate: 44100,
+            bits_per_sample: 16,
+            sample_format: hound::SampleFormat::Int,
+        };
+        let mut writer =
+            WavWriter::create(input_wav_path, spec).expect("Failed to create test input WAV file");
+        for t in (0..44100).map(|x| x as f32 / 44100.0) {
+            let sample = (t * 440.0 * 2.0 * std::f32::consts::PI).sin();
+            let amplitude = i16::MAX as f32;
+            writer.write_sample((sample * amplitude) as i16).expect("Failed to write sample");
+        }
+        writer.finalize().expect("Failed to finalize test input WAV file");
```

Comment:
input.wav was missing in main and I got some inexplicable format issues with it in certain CI environments when I added it back (but it worked locally). So instead I'm creating the file on the fly for the test @theomonnom